src/components/@molecules/TransactionDialogManager/TransactionDialogManager.tsx

```diff
@@ -101,7 +101,7 @@ export const TransactionDialogManager = ({
   }, [selectedKey, selectedItem, onDismiss, dispatch, t])
 
   return (
-    <Dialog variant="blank" open={!!state.selectedKey} onDismiss={onBackgroundDismiss}>
+    <Dialog variant="blank" open={!!selectedKey} onDismiss={onBackgroundDismiss}>
```

Comment:
pretty sure it was previously `state.selectedKey` to make sure that that dismiss animation is finished before unloading the modal